File: frontend/src/models/roomModels/roomReadingModels/ServerSendsTemperatureReadingsForRoom.ts
import {SensorDto} from "../../Entities";
import {BaseDto} from "../../baseDto";


export class ServerSendsTemperatureReadingsForRoom extends BaseDto<ServerSendsTemperatureReadingsForRoom> {
  RoomId!: number;
  TemperatureReadings!: SensorDto[];
}

File: backend/api/serverEventModels/roomDtos/ServerSendsTemperatureReadingsForRoom.cs
using infrastructure.Models;
using lib;

namespace api.serverEventModels.roomDtos;

public class ServerSendsTemperatureReadingsForRoom : BaseDto
{
    public int RoomId { get; set; }
    public IEnumerable<SensorDto>? TemperatureReadings { get; set; }
}